### DISCUSSION TITLE
Private Api

### DISCUSSION THREAD

**Discussion by mickael-fd** [brefphp/bref#1434] [General]
Bref.sh use http api through aws api gateway but only REST api can be private.
i use serverless apps with Symfony Framework for internal project and i don't want to publish them into internet network only on my vpn.
Any solutions ?
thank you

**Comment by mnapoli** [comment]
You can read how to deploy REST APIs here: https://www.serverless.com/framework/docs/providers/aws/events/apigateway But I've never done private APIs in VPC, so you  are on your own after that.

**Comment by deleugpn** [comment]
In case you're interested in a simple and cheap approach: https://blog.deleu.dev/using-aws-lambda-as-a-private-laravel-api/
